Sejima/Source/MUGraphView/GraphSegment.swift

```diff
@@ -0,0 +1,102 @@
+/*
+ Copyright (C) 2016 Apple Inc. All Rights Reserved.
+ See LICENSE.txt for this sample’s licensing information
+ 
+ Abstract:
+ A `UIView` subclass that represents a segment of data in a `GraphView`.
+ */
+
+import UIKit
+import simd
+
+/// Apple's GraphSegment
+internal final class GraphSegment: UIView {
+
+    // MARK: - Properties
+
+    /// All values for this segments.
+    private(set) var dataPoints = [double3]()
```

Comment:
We shall have a type alias to avoid double3 type

Reply:
MUGraphView doesn't import simd, but still in Apple's code in GraphView and GraphSegment.